### ISSUE TITLE
Remove error-polyfill

### ISSUE THREAD

**Issue by Jack-Works** [near/near-api-js#833]
Hi! Please do not add a polyfill in a library. It throws an error in our environment.

[https://user-images.githubusercontent.com/5390719/161533892-eeab145e-1f6b-4912-8a6e-4f87755054be.png](https://user-images.githubusercontent.com/5390719/161533892-eeab145e-1f6b-4912-8a6e-4f87755054be.png))

**Comment by esaminu** [comment]
Hi @Jack-Works, thanks for reporting! Could we please have more info regarding your environment here and any other steps to repro?

**Comment by Jack-Works** [comment]
Open `http://example.com`, then open console.

Input:

```js
await import('https://cdn.jsdelivr.net/npm/ses/dist/ses.mjs')
// This step will freeze window.Error
lockdown()

import('https://cdn.jsdelivr.net/npm/near-api-js@1.0.0/dist/near-api-js.js').catch(console.error)
// TypeError#1: Cannot add property getStackTrace, object is not extensible
```

![image](https://user-images.githubusercontent.com/5390719/187012173-a868520c-b67e-41d5-b3ba-ac1dfcd6a846.png)

**Comment by marcinbodnar** [comment]
The reason this error appears in the example is because we're using `error-polyfill` to implement stack trace. But I'm not sure how we want to handle this issue. @esaminu do you have a clue?

**Comment by vikinatora** [comment]
This is resolved by #1224.
